kernel/packages/decentraland-loader/lifecycle/manager.ts

```diff
@@ -101,7 +101,7 @@ export async function initParcelSceneWorker() {
     metaContentService: DEBUG
       ? resolveUrl(document.location.origin, '/local-ipfs')
       : getFetchMetaContentService(window.globalStore.getState()),
-    contentServerBundles: DEBUG ? '' : getServerConfigurations().contentAsBundle + '/',
+    contentServerBundles: DEBUG || PIN_CATALYST ? '' : getServerConfigurations().contentAsBundle + '/',
```

Comment:
Not sure about precedence here, is this the same as 
```suggestion
    contentServerBundles: (DEBUG || PIN_CATALYST) ? '' : getServerConfigurations().contentAsBundle + '/',
```
?

Reply:
Indeed, it's the same result.